lib/grape-swagger/endpoint.rb

```diff
@@ -198,7 +198,7 @@ def response_object(route)
         response_model = expose_params_from_model(value[:model]) if value[:model]
 
         if route.request_method == 'DELETE' && !value[:model].nil?
```

Comment:
Sorry, I'm not sure what you mean, do you want me to revert that change?  That's part of the change that introduced the regression.  But I think that altered behavior intentionally, correct?  If not, then I can revert that change, and I think my new unit tests will still pass; however that means that the new behavior introduced by that line will be reverted back to what it was... is that acceptable?  (sorry if i'm misunderstanding you!)